test/extended/baremetal/hosts.go

```diff
@@ -147,7 +147,7 @@ var _ = g.Describe("[sig-installer][Feature:baremetal][Serial] Baremetal platfor
 		helper.DeleteAllExtraWorkers()
 	})
 
-	g.It("skip inspection when disabled by annotation", func() {
+	g.It("skip inspection when disabled by annotation [Timeout:20m]", func() {
```

Comment:
@bfournie I think this test it's ok, it's the scale&grow that needs to be increased

Reply:
> @bfournie looks like a typo, it's the scale&grow that needs to be increased

Yeah that's what the test results look like. Updated timeout in scale and grow.